cmd/load-runner/docker_test.go

```diff
@@ -0,0 +1,516 @@
+package main
```

Comment:
[optional] You got a nice coverage here although none of these tests seem to be using a real docker compose setup. I'd find it more useful in this specific case instead of mocking.

Reply:
Tried to tackle the docker compose setup but takes some time. Added TODO so that we spend some proper time to make this change.

Reply:
@pChondros have a look here: https://github.com/rudderlabs/compose-test